intuita-webview/scripts/build-react-no-split.ts

```diff
@@ -29,3 +22,15 @@ config.output.filename = 'static/js/[name].js';
 // Rename main.{hash}.css to main.css
 config.plugins[5].options.filename = 'static/css/[name].css';
 config.plugins[5].options.moduleFilename = () => 'static/css/main.css';
+
+const appDirectory = fs.realpathSync(process.cwd());
+const resolveApp = (relativePath) => path.resolve(appDirectory, relativePath);
+
+// @TODO fix ManifestPlugin so it dont break build with custom entrypoints
+// as quick fix just removed it
+config.plugins.splice(6, 1);
```

Comment:
do we need the manifest files for the extension?

Reply:
Probably we can just exclude it

Reply:
If it works without it (as in, if the Extension Markeplace allows it), I think it's fine to exclude it.